tests/dbgp/dbgpclient.php

```diff
@@ -1,16 +1,53 @@
 <?php
 class DebugClient
 {
+	protected $triedPorts = 0;
+
+	protected $startPort = 9991;
```

Comment:
Again a long time ago now, but what I remember is that in instances where the port wasn't released other tests would fail because the port was in use. It's mostly a fail safe thing so that one test would not instantly fail all other tests. I agree that it's not the most elegant thing, but it did what it was supposed to do. It only adds some functionality in a manner that should not impact anything else.